bouncer/bouncer.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/sam/build-python3.9:latest
+
+## Initial setup
+
+COPY ./bouncer ${LAMBDA_TASK_ROOT}/
```

Comment:
This will trigger all dependency installation after every minor change in the code. 
The correct way is:
1. Copy the requirements.txt 
2. Install the requirements
3. Copy the code
4. Launch the code